pkg/rpc/response/result/version.go

```diff
@@ -4,7 +4,7 @@ type (
 	// Version model used for reporting server version
 	// info.
 	Version struct {
-		Port      uint16 `json:"tcp_port"`
+		Port      uint16 `json:"tcpPort"`
```

Comment:
I don't think it's enough, this structure needs to be compatible with all nodes and some will send us `ws_port` that we should be able to represent correctly on the client (even though our server never sends it).

Reply:
And where this `tcpPort` comes from? I see `tcp_port` in the `master` code, am I missing something?

Reply:
`tcpPort` and `useragent` were taken from specification (https://docs.neo.org/v3/docs/en-us/reference/rpc/latest-version/api/getversion.html). Now replaced by `tcp_port` and `user_agent`.

Reply:
Wow. So there is a difference between implementation and documentation? Please file a bug for neo-modules then. We'll use implementation's field names for now.